docs/kiwix-serve.rst

```diff
@@ -375,10 +375,20 @@ specified ZIM file.
   ``content``: :term:`name <ZIM name>` of the ZIM file.
 
 
-``/raw``
---------
+``/raw/ZIMID/content/PATH/IN/ZIMFILE``
+--------------------------------------
 
-Blablabla
+Returns the entry with path ``PATH/IN/ZIMFILE`` from the ZIM file with
+:term:`name <ZIM name>` ``ZIMID``. This endpoint almost duplicates (with some
+subtle technical differences) the newer endpoint
+`/content/ZIMID/PATH/IN/ZIMFILE`_ and exists mostly for historical reasons.
```

Comment:
This almost duplicates `/content` but not for historical reasons.
`/content` is intended to be use to browse the zim file. **For now** the only difference is that we redirect `/content/ZIMID/` to the main page. But it is the current version.

On the contrary `/raw` guaranty that we return the raw content in the zim file and only this, without any modification. Now and in the future. It can be used by script that what to check thing on zim files.

Reply:
Fixed